pages/about.html

```diff
@@ -1,12 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About me</title>
+    <link href="../styles/style.css" rel="stylesheet" />
+    <link href="../styles/about.css" rel="stylesheet" />
+  </head>
+  <body>
+    <nav>
+      <div class="logo">
+        <img src="/assets/logo.png" alt="Website-logo" />
+      </div>
+      <ul>
+        <li>
+          <a href="index.html"><em>00.</em>Home</a>
+        </li>
+        <li>
+          <a href="about.html"><em>01.About</em></a>
+        </li>
+        <li>
+          <a href="portfolio.html"><em>02.</em>Projects</a>
+        </li>
+        <li>
+          <a href="contact.html"><em>03.</em>Contact</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="contact-container">
+      <ul class="contact-list">
+        <li>
+          <a href="https://github.com/diakymenko" target="_blank">GitHub</a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/daria-iakymenko-26177831"
+            target="_blank"
+            >LinkedIn</a
+          >
+        </li>
+        <li>
+          <a href="mailto:daria.iakymenko@gmail.com">Email</a>
+        </li>
+      </ul>
+    </div>
+    <div class="main">
+      <header>About me</header>
+      <h1><span>My coding journey</span></h1>
+      <div class="wrapper">
+        <div class="left-inner">
+          Hello! My name is Daria Iakymenko and currently I’m learning software
+          development at
+          <a href="https://adadevelopersacademy.org/" target="_blank"
+            ><em>Ada Developers Academy</em></a
+          >. My interest in programming sparkled when I was trying to optimize a
+          routine procedure and realized how important coding skills could be!
+          <br />
+          <br />
+          After months of self-learning I was honored to be accepted to Ada Dev
+          Academy that gave me an opportunity to learn CS fundamentals,
+          Backend/Frontend and continuously improve in my work.
+          <br />
+          <br />
+          Here are a few technologies I’ve been working with recently:
```

Comment:
To make this more semantic, consider wrapping each paragraph in a `<p>` tag. Then, you ccan control the spacing between them using CSS instead of `<br />`. This makes things a bit more flexible!